packages/providers/lib.esm/base-provider.d.ts

```diff
@@ -139,7 +128,6 @@ export declare class BaseProvider extends Provider implements EnsProvider {
     getTransactionReceipt(transactionHash: string | Promise<string>): Promise<TransactionReceipt>;
     getLogs(filter: Filter | FilterByBlockHash | Promise<Filter | FilterByBlockHash>): Promise<Array<Log>>;
     getEtherPrice(): Promise<number>;
```

Comment:
We should rename this method to `getHbarPrice` and throw not implemented for now